elementary/clients/slack/client.py

```diff
@@ -130,7 +131,12 @@ def _get_channels(self) -> List[dict]:
 
     def _handle_send_err(self, err: SlackApiError, channel_name: str) -> bool:
         err_type = err.response.data['error']
-        if err_type == 'not_in_channel':
+        if err_type == 'ratelimited':
```

Comment:
Seems like you can get rate limit also for webhooks - 
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/4517234/188872575-ff6d3800-69b2-4a82-b969-b2c9a9c0b764.png">


I think we should check the http error code and extract the retry after from the headers

Reply:
They are saying you can, but I run it with an endless loop and got none.
I know that we can try and extract the header but the response is in a different format, and I am not sure how to extract it because I haven't been able to produce this error.

Reply:
you don't need to reproduce - it's simply reading the status code and extracting it from the headers in the response (according to the spec in their docs)